templates/_templating_scripting.py

```diff
@@ -211,6 +212,10 @@ def create_issue(title: str, body: str) -> None:
         )
         if any(issue["title"] == title for issue in existing_issues):
             return
+        if author in BOT_AUTHORS:
+            # if the author is a bot, we don't want to assign the issue to the bot,
+            # so instead choose a human author from the latest commit
+            author = get_commit_authors(commits[-1])[0]
```

Comment:
I don't see why this can't **also** be a bot  
  - doesn't this need a retry and/or backstop approach ?

Reply:
Unfortunately I don't think I'm experienced enough in this area to be safe reviewing this !